file/readfile.go

```diff
@@ -132,3 +140,26 @@ func configFilesInDir(dir string) ([]string, error) {
 	}
 	return res, nil
 }
+
+func processEnvs(content string) (string, error) {
+	const envVarPrefix = "DECK_"
+	t := template.New("state").Funcs(template.FuncMap{
+		"env": func(key string) (string, error) {
+			if !strings.HasPrefix(key, envVarPrefix) {
+				return "", fmt.Errorf("environment variables in the state file must "+
+					"be prefixed with 'DECK_', found: '%s'", key)
+			}
+			return os.Getenv(key), nil
+		},
```

Comment:
For readability and maintainability, please extract the lambda function into a named function, for example `func GetPrefixedEnvVar`

Reply:
Done.